### PR TITLE
feat(selection): support scroll wheel to move dragging blocks

### PR DESCRIPTION
**Pull request by fundon**
https://user-images.githubusercontent.com/27926/230816412-4d19fa98-e633-4966-b5f5-544329351104.mov

### CODE REVIEW

**Comment by fundon** [comment]
found some issues

File: packages/blocks/src/components/drag-handle.ts [hunk 1]
```diff
@@ -474,6 +482,104 @@ export class DragHandle extends WithDisposable(LitElement) {
     );
   }
 
+  static clacTarget(
```
Comment (doodlewind) [inline packages/blocks/src/components/drag-handle.ts:485]: Is this a typo of `calcTarget`? Also, having the return value of this function typed will look better, current implementation looks a bit ad-hoc.

**Comment by fundon** [comment]
Included #1440

**Comment by fundon** [comment]
Included #1310

**Comment by fundon** [comment]
Included #1164

**Review by doodlewind — APPROVED** [review]
I suggest having some further refactoring on selection for decoupling (making the selection-related components pluggable for the editor) and better reusability, since its complexity grows fast. Will discuss with @fundon for more plannings, thanks!

**Pull request merged**